README.md

```diff
@@ -42,9 +42,9 @@ Spartacus provides core storefront features such as:
 # Requirements
 
 - SAP Commerce Cloud instance (Release 1808)
```

Comment:
Should the version be 1811? Or both 1808 and 1811?

Reply:
Good catch.  As far as I know, for now, we support the latest version of SAP Commerce.